src/components/RiskComponentDropdown.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <v-expansion-panel class="riskComponentDropdown">
+    <v-expansion-panel-header class="d-flex flex-row">
+      <div :class="riskClass" class="componentIconContainer" flex-shrink-1>
+        <CrowdingIcon v-if="isCrowding" class="componentIcon" />
+        <DropletsIcon v-if="isDroplets" class="componentIcon" />
+        <TimeIcon v-if="isTime" class="componentIcon" />
+        <VentIcon v-if="isVent" class="componentIcon" />
```

Comment:
it may be better to use the `:is` binding here to prevent repeating code—the icon could be computed in RiskComponents and passed as a prop

Reply:
I'm re-structuring the data to make this a bit more friendly, so that "risk characteristics" are structured into a hash instead of flattened out in the data -- that will make your suggestion even more pertinent. I think good to merge the current version though.